### ISSUE TITLE
Handle status=AlreadyTracked

### ISSUE THREAD

**Issue by fast-slow-still** [microsoft/MixedReality-WorldLockingTools-Samples#56]
On downloading a cloud anchor the second time within a session, the anchor's status will be AlreadyTracked (guessing here, it's not explicitly documented). Currently, that means the anchor will be ignored.

Needs more consideration, but it seems like the proper behavior would be something like:

```
If status==AlreadyTracked
    If anchor is in List<AnchorRecords> records
         use that AnchorRecord
    else
         add it as if status==Located
```

**Comment by fast-slow-still** [comment]
Some explanation of different LocateAnchorStatus states [here](https://github.com/Azure/azure-spatial-anchors-samples/issues/100).

**Comment by fast-slow-still** [comment]
Addressed as part of PR #63 , using cached records if available as suggested above. See ~line696 in function DownloadRecord in PublisherASA.cs.
